api/app/Models/TestEntity.php

```diff
@@ -39,6 +39,16 @@ class TestEntity extends BaseModel {
     ];
 
     /**
+     * Get the access route for the entity.
+     *
+     * @return string
+     */
+    public function entityRoute()
+    {
+        return '/test/entities';
+    }
+
+    /**
```

Comment:
This is to be able to create the self attribute, from the model factory, we will not be able to determine the route used to access the model?